### PR TITLE
Fix tor bridge nickname

### PR DESCRIPTION
**Pull request by nopdotcom**
For nicknames, we need 1-19 characters from `[A-Za-z0-9]`. Our wordlist has a bunch of 8-character words, so we're limited to two words for this. **TitleCase** the words for moderately better recognition. Chop at 18 characters in case our wordlist gets expanded.

### CODE REVIEW

File: playbooks/roles/tor-bridge/tasks/main.yml [hunk 1]
```diff
@@ -27,8 +27,7 @@
 - import_tasks: firewall.yml
 
 - name: Generate a random Nickname for the bridge
-  shell: grep -v -P "[\x80-\xFF]" /usr/share/dict/american-english-huge | sed -e "s/'//" | shuf -n 2 | xargs | sed -e 's/ //' | cut -c 1-16 > {{ tor_bridge_nickname_file }}
-  args:
+  shell: "{{ streisand_word_gen.tor_nickname }} > {{ tor_bridge_nickname_file }}  args:
```
Comment (cpu) [inline playbooks/roles/tor-bridge/tasks/main.yml]: You lost a `"` at the end of the `shell:` item and pulled up the `args:` item here. It's breaking CI: 
```
ERROR! Syntax Error while loading YAML.
The error appears to have been in '/home/travis/build/StreisandEffect/streisand/playbooks/roles/tor-bridge/tasks/main.yml': line 31, column 15, but may
be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:
  shell: "{{ streisand_word_gen.tor_nickname }} > {{ tor_bridge_nickname_file }}  args:
    creates: "{{ tor_bridge_nickname_file }}"
              ^ here
```

Comment (nopdotcom) [reply playbooks/roles/tor-bridge/tasks/main.yml]: Thanks. This is motivating me to turn on all the YAML validation stuff in my editor. Right now the validator is unhappy with long lines, but I think I'll turn that off.

**Comment by nopdotcom** [comment]
Tor tested as working on a deploy of this version.

Pretty sure Emacs and focus issues is what did me in last time.

**Comment by cpu** [comment]
> Pretty sure Emacs and focus issues is what did me in last time.

just add vim

**Pull request merged**